DXRModules/DeusEx/Classes/DXRPlayerStats.uc

```diff
@@ -94,7 +94,7 @@ function CapHealth(out int health, int d)
 
 simulated function FirstEntry()
 {
-    if(dxr.dxInfo.missionNumber == 0) {
+    if(class'DXRStartMap'.static._IsStartMap(dxr) || dxr.dxInfo.missionNumber == 0) {
```

Comment:
missionNumber 0 is training btw, the intro (despite its filename) is mission 98, maybe we should try this

```suggestion
    if(dxr.dxInfo.missionNumber == 0 || dxr.dxInfo.missionNumber == 98) {
```

Reply:
I tried checking for `dxr.localURL ~= "INTRO"` like you suggested. It was executing the code, it just didn't fix the problem.

Reply:
ok yea intro would've caught it too

Reply:
I think we should also have a check for hardcore, which is `flags.autosave==5`

Reply:
Why should ironman be different?

Reply:
it's supposed to be play until you die, especially WaltonWare Ironman

Reply:
Makes sense then.